packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -39,17 +40,18 @@ import {
 } from './utils';
 
 import * as events from 'events';
-import * as grpcModule from 'grpc';
+import * as grpcTypes from 'grpc';
 import * as shimmer from 'shimmer';
 import * as path from 'path';
 
 /** The metadata key under which span context is stored as a binary value. */
 export const GRPC_TRACE_KEY = 'grpc-trace-bin';
 
-let grpcClientModule: object;
+let grpcClientModule: GrpcInternalClientTypes;
 
 export class GrpcPlugin extends BasePlugin<grpc> {
   static readonly component = 'grpc';
+  readonly supportedVersions = ['^1.23.3'];
```

Comment:
Could you please update GRPC plugin README about supported version note?

Reply:
Updated it here https://github.com/open-telemetry/opentelemetry-js/pull/325/files#diff-14f6f7ff20cfa54c02fe63c8d1c8f344L7-R7